jujugui/static/gui/src/app/components/deployment/summary/change-item/change-item.js

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
Note for reviewer: this component is a duplicate of the existing change-item component.